drivers/keyboard.c

```diff
@@ -0,0 +1,93 @@
+/* ------------------------------------------------------------------------ *\
+**
+**  This file is part of the Chaos Kernel, and is made available under
+**  the terms of the GNU General Public License version 2.
+**
+**  Copyright (C) 2017 - Benjamin Grange <benjamin.grange@epitech.eu>
+**
+\* ------------------------------------------------------------------------ */
+
+#include <kernel/init.h>
```

Comment:
Ordering for includes is:

* chaosdef.h/chaoserr.h
* kernel/*.h
* arch/x86/*.h
* driver/*.h
* libc header

Reply:
done.